app/controllers/katello/api/v2/root_controller.rb

```diff
@@ -8,6 +8,19 @@ class Api::V2::RootController < Api::V2::ApiController
     end
 
     def resource_list
```

Comment:
I thought this endpoint was used by hammer but now I'm not so sure. Any ideas?

Reply:
i don't see this in foreman at all, and looking at history it was moved over from the v1 api, so i imagine it can be removed?